docs/src/main/asciidoc/_testsuite/WildFly_Testsuite_Harness_Developer_Guide.adoc

```diff
@@ -15,7 +15,7 @@ merged here later.
 == Adding a new maven plugin
 
 The plugin version needs to be specified in jboss-parent. See
-https://github.com/jboss/jboss-parent-pom/blob/master/pom.xml#L65 .
+https://github.com/jboss/jboss-parent-pom/blob/main/pom.xml#L65 .
```

Comment:
Again, remove the line number and describe the paragraph a bit differently so the user can understand what the links provide, for example, The plugin version needs to be specified in jboss-parent at the `<properties>` section of the jboss-parent pom.xml file